templates/taxi/car_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create"}} car</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit" />

```

Comment:
According to checklist, not bad will be add Cancel button apart from Delete one.